src/hooks/useMixpanelPortfolioTracking/useMixpanelPortfolioTracking.tsx

```diff
@@ -29,16 +30,22 @@ export const useMixpanelPortfolioTracking = () => {
   const isMarketDataLoaded = useAppSelector(selectMarketDataLoaded)
 
   useEffect(() => {
+    // only track anonymized portfolio for real wallets
     if (isDemoWallet) return
     // we've changed wallets, reset tracking
     if (!wallet) return setIsTracked(false)
     // only track once per wallet connection
     if (isTracked) return
     // only track if market data is loaded
-    if (isMarketDataLoaded && !isTracked) {
-      getMixPanel()?.identify()
-      getMixPanel()?.people.set(anonymizedPortfolio)
-      setIsTracked(true)
-    }
+    if (!isMarketDataLoaded) return
+
+    const mp = getMixPanel()
+    if (!mp) return
+    // track portfolio loaded event now that market data is loaded
+    mp.track(MixPanelEvents.PortfolioLoaded, anonymizedPortfolio)
```

Comment:
this is async because we need all the market data to give anonymized portfolio values